test/test_time_parser.rb

```diff
@@ -0,0 +1,195 @@
+require_relative 'helper'
+require 'fluent/test'
+require 'fluent/time'
+
+class TimeParserTest < ::Test::Unit::TestCase
+  def setup
+    Fluent::Test.setup
+  end
+
+  def test_call_with_parse
```

Comment:
I have no idea: it's just moved from `test/plugin/test_parser_time.rb` to there.